vl53v1x/vl53l1x.go

```diff
@@ -0,0 +1,541 @@
+// Package vl53l1x provides a driver for the VL53L1X time-of-flight
+// distance sensor
+//
+// Datasheet:
+// https://www.st.com/resource/en/datasheet/vl53l1x.pdf
+// This driver was based on the library https://github.com/pololu/vl53l1x-arduino
+// and ST's VL53L1X API (STSW-IMG007)
+// https://www.st.com/content/st_com/en/products/embedded-software/proximity-sensors-software/stsw-img007.html
+package vl53l1x
+
+import (
+	"machine"
+	"time"
+)
+
+type DistanceMode uint8
+type RangeStatus uint8
+
+type RangingData struct {
```

Comment:
This struct seems to be used internally only, so I don't think it should be exported.